central/debug/service/service.go

```diff
@@ -497,6 +498,8 @@ type debugDumpOptions struct {
 }
 
 func (s *serviceImpl) writeZippedDebugDump(ctx context.Context, w http.ResponseWriter, filename string, opts debugDumpOptions) {
+	subCtx, cancel := context.WithTimeout(ctx, debugDumpHardTimeout)
```

Comment:
nit: I'd prefer something more descriptive such as `debugDumpCtx` instead of `subCtx`.